yarn.lock

```diff
@@ -202,13 +203,12 @@
     widest-line "^2.0.1"
     wrap-ansi "^4.0.0"
 
-"@oclif/plugin-help@^3":
-  version "3.1.0"
-  resolved "https://registry.yarnpkg.com/@oclif/plugin-help/-/plugin-help-3.1.0.tgz#507ff8e9cabe734672b12d1ec0b79812d18c58a8"
-  integrity sha512-orSWpXGlJaX16eSjAtI8scA8QhrjQOaCSHodEx52t18JKbIVzG8jcngugyWAOB/V4jhPl0rdiVk9XFsaIIiG2g==
+"@oclif/plugin-help@^2.1.6":
```

Comment:
Note that there are two versions of this in the lock file. So if we upgrade per the above comment we either need a selective resolution to resolve to the latest, or to also update whatever is requesting the older version.